src/stmt.cpp

```diff
@@ -3657,6 +3650,44 @@ void AssertStmt::EmitCode(FunctionEmitContext *ctx) const {
     free(errorString);
 }
 
+void AssertStmt::EmitAssumeCode(FunctionEmitContext *ctx, const Type *type) const {
+
```

Comment:
Typically we don't have extra vertical space in the beginning of the function.
```suggestion
```

Reply:
Done.